src/entities/Report.ts

```diff
@@ -0,0 +1,42 @@
+import { Entity, Column, OneToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { IsDate, MaxLength, IsString } from 'class-validator';
+import { User } from './User';
+
+export interface APIReport {
+	id: string;
+	reportingUserID: string;
+	currentTime: string;
+	description: string;
+}
+
+@Entity()
+export default class Report {
+	@PrimaryGeneratedColumn('uuid')
+	public id!: string;
+
+	@OneToOne(() => User, user => user.report)
```

Comment:
```suggestion
	@ManyToOne(() => User)
```

A user can have more than one report against them (so many reports can share one user) which makes this a ManyToOne relation instead.